### ISSUE TITLE
job execution status always set to SUCCESS

### ISSUE THREAD

**Issue by leos** [tokern/data-lineage#88]
As per the lines below, `job_execution.status` will always be `JobExecutionStatus.SUCCESS`. Is this intentional?

https://github.com/tokern/data-lineage/blob/master/data_lineage/server.py#L311-L313

**Comment by vrajat** [comment]
Yes :) I havent implemented a feature to use status yet. I put it in the data model to future proof it.
